src/test/resources/test.properties

```diff
@@ -4,6 +4,15 @@ spring.profiles.active=test
 # Liquibase Database Migration
 liquibase.contexts=test
 
+#######################################################################################################################
+# Application
+#######################################################################################################################
+app.name=Voyage
+app.contact-support.email=support@LighthouseSoftware.com
+app.contact-support.phone=651-888-6021
+app.contact-support.website=http://LighthouseSoftware.com/support
+app.default-country=US
+
 # Voyage Security: Test client for integration tests
```

Comment:
These props are getting duplicated from application.properties - by default these should come from application, unless the user wants to override them for tests